packages/primitives/src/index.js

```diff
@@ -8,6 +8,12 @@ export const Note = 'NOTE';
 export const Path = 'PATH';
 export const Rect = 'RECT';
 export const Line = 'LINE';
+export const Form = 'FORM';
+export const FormField = 'FORM_FIELD';
```

Comment:
Is this necessary? I get how you can create hierarchical structures but wonder it it's useful at all to expose this